composer.json

```diff
@@ -17,7 +17,7 @@
     },
     "require-dev": {
         "athletic/athletic": "~0.1",
-        "phpunit/PHPUnit": "~4.0",
+        "phpunit/phpunit": "^5.7.25",
```

Comment:
Hm... not so easy, need more work to update PHPUnit ...

Reply:
Wow, actually I'm so surprised, tests pass ! We must have there only basic tests then:)

Reply:
:) One thing to do (in next pr) it's to add support for phpunit 6.